scripts/update-niv.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -p git jq gitAndTools.hub niv -i bash
+
+set -euo pipefail
+
+author="serokell-bot <operations+github@serokell.io>"
+
+pkgs() {
+    jq ".[] | .repo, .rev" nix/sources.json | tr -d \" | xargs -n2 echo
```

Comment:
I really like documenting these opaque chains of commands for future reference

Reply:
Writing them is much easier than reading them.